.github/workflows/publish-testpypi.yml

```diff
@@ -8,8 +8,8 @@ on:
     - cron: "0 0 * * *"
 
 jobs:
-  deploy:
-    if: github.repository_owner == 'unitaryfund'
+  build:
+    if: github.repository_owner == 'unitaryfund' && github.event_name == 'push'
```

Comment:
Since this is on a schedule, where is event_name `push`?

Reply:
Ahh good catch. I was reading a document on securing github actions workflows and I blindly applied this recommendation without much thinking!

Reply:
Removed in https://github.com/unitaryfund/mitiq/pull/2320/commits/c8abc2190b1d9cad346249087345000cea572ade.